# end the initalization by canceling or by saying ok

string  message
---
bool    success
